minigalaxy/ui/properties.py

```diff
@@ -0,0 +1,96 @@
+import gi
+import os
+import subprocess
+import webbrowser
+
+gi.require_version('Gtk', '3.0')
+
+from gi.repository import Gtk, GLib
+from minigalaxy.paths import UI_DIR, THUMBNAIL_DIR
+from minigalaxy.translation import _
+from minigalaxy.launcher import config_game
+
+@Gtk.Template.from_file(os.path.join(UI_DIR, "properties.ui"))
+class Properties(Gtk.Dialog):
+    __gtype_name__ = "Properties"
+    gogBaseUrl = "https://www.gog.com"
+
+    image = Gtk.Template.Child()
+    button_properties_cancel = Gtk.Template.Child()
+    button_properties_ok = Gtk.Template.Child()
+    button_properties_support = Gtk.Template.Child()
+    button_properties_store = Gtk.Template.Child()
+    button_properties_open_files = Gtk.Template.Child()
+    button_properties_settings = Gtk.Template.Child()
+    switch_properties_show_fps = Gtk.Template.Child()
+    entry_properties_variable = Gtk.Template.Child()
+    entry_properties_command = Gtk.Template.Child()
+
+    def __init__(self, parent, game, api):
+        Gtk.Dialog.__init__(self, title=_("Properties of {}".format(game.name)))
+        self.parent = parent
+        self.game = game
+        self.api = api
+
+        #Show the image
+        self.load_thumbnail()
+
+        # Disable/Enable buttons
+        self.button_sensitive(game)
+
+        # Retrieve variable & command each time Properties is open
+        self.entry_properties_variable.set_text(self.game.get_info("variable"))
+        self.entry_properties_command.set_text(self.game.get_info("command"))
+
+        # Keep switch FPS disabled/enabled
+        self.switch_properties_show_fps.set_active(self.game.get_info("show_fps"))
+
+    @Gtk.Template.Callback("on_button_properties_cancel_clicked")
+    def cancel_pressed(self, button):
+        self.destroy()
+
+    @Gtk.Template.Callback("on_button_properties_ok_clicked")
+    def ok_pressed(self, button):
+        self.game.set_info("variable", str(self.entry_properties_variable.get_text()))
+        self.game.set_info("command", str(self.entry_properties_command.get_text()))
+        self.game.set_info("show_fps", self.switch_properties_show_fps.get_active())
```

Comment:
please make sure all self.game.set_info() is under:
if game.is_installed():

Reply:
Do you think it's necessary because when the game is not installed, variable/command text bar and show_fps switch are disabled for the user so he can't use them.

Reply:
Yep. Clicking "OK" will call self.game.set_info() (even if variable is empty). This will probably create minigalaxy-info.json file in current working directory.

You can alternatively alter game.set_info() method itself to be executed only on installed games.